README.md

```diff
@@ -5,15 +5,15 @@
 
 # Odyssey Design System
 
-This repository contains the code for Okta's Design System Odyssey. It includes Icons, SCSS, and React Components used to build products and experiences at Okta.
+This repository contains the code for Okta's Design System Odyssey. It includes Icons and React Components used to build products and experiences at Okta.
 
 ## Packages
 
-| Package/README                                                                                      | Description                                                        |
-| --------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ |
-| [@okta/odyssey-icons](https://github.com/okta/odyssey/tree/master/packages/odyssey-icons/README.md) | Odyssey SVG icons.                                                 |
-| [@okta/odyssey-react](https://github.com/okta/odyssey/tree/master/packages/odyssey-react/README.md) | Odyssey components for [React](https://reactjs.org/).              |
-| [@okta/odyssey-scss](https://github.com/okta/odyssey/tree/master/packages/odyssey-scss/README.md)   | Odyssey SCSS, includes components, includes base and reset styles. |
+| Package/README                                                                                                      | Description                                                      |
+| ------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| [@okta/odyssey-design-tokens](https://github.com/okta/odyssey/tree/master/packages/odyssey-design-tokens/README.md) | Low level variables used to describe component style properties. |
```

Comment:
I've tried to become more careful about my use of the following vocabulary: `variables` vs `tokens` vs `properties`. In this case, we're providing tokens that don't describe component style properties, but rather global style values to be consumed in a variety of formats.

We can take care of this in a future PR

Reply:
Fixed.